app/forms/SubmitVatReturnForm.scala

```diff
@@ -32,8 +32,12 @@ object NineBoxForm {
       "box6" -> bigDecimal,
       "box7" -> bigDecimal,
       "box8" -> bigDecimal,
-      "box9" -> bigDecimal
-    )(NineBoxModel.apply)(NineBoxModel.unapply)
+      "box9" -> bigDecimal,
+      "flatRateScheme" -> boolean,
```

Comment:
These values don't necessarily need to be in the form and as hidden fields on the view - they could just be added to session separate to the 9 box values

Reply:
Agreed to reduce complexity of code to retain in form